src/egon/data/datasets/chp_etrago.py

```diff
@@ -107,6 +108,12 @@ def insert():
     chp_el["link_id"] = range(
         db.next_etrago_id("link"), len(chp_el) + db.next_etrago_id("link")
     )
+
+    # Add marginal cost
+    chp_el["marginal_cost"] = get_sector_parameters("gas", "eGon2035")[
+        "marginal_cost"
+    ]["CH4"]
+
```

Comment:
The fuel costs for CH4 are already added to the CH4-feedin generators. When you add these costs here, they are assigned twice.

Reply:
I changed it to only consider VOM for gas CHPs